docs/conversation.md

```diff
@@ -453,6 +458,11 @@ Get all (for moderators and in case of "free selection") or the assigned breakou
         + `403 Forbidden` When the current user is not a moderator, owner or guest moderator
         + `404 Not Found` When the conversation could not be found for the participant
 
+	- Data: See array definition in `Get user´s conversations`
+
+## Get conversations avatar (binary)
+
+* ⚠️ Preview - Might be modified before the capability is added
 * Required capability: `avatar`
 * Method: `GET`
 * Endpoint: `/room/{token}/avatar`
```

Comment:
Shouldn't we mention `?v=<avatarVersion>` query param here for HTTP Cache?

Reply:
Query parameters are not part of the API, in fact the API does not eve handle the parameter, so you can not get an older version by specifying an older parameter or something a like, so I'd like not to add it to the docs

Reply:
It is not a part of API, but developers should use it. And I'm sure somebody will miss it.

Maybe we can add a note?

In the previous version, an example of the get image UTL (in POST avatar) contained `?v=<avatarVersion>`.